.github/workflows/merge-staging.yml

```
@@ -0,0 +1,30 @@
+name: "merge staging(-next)"
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Merge every 6 hours
+    - cron:  '* */6 * * *'
+
+jobs:
+  sync-branch:
+    if: github.repository == 'NixOS/nixpkgs'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

```suggestion
      - uses: actions/checkout@v2
```
Upstream seems to recommend pinning this in their examples. https://github.com/actions/checkout